### ISSUE TITLE
Error on ImageNet training: the following arguments are required: DIR

### ISSUE THREAD

**Issue by ZangHuanyu** [pytorch/examples#788]
As the title, I'm learning how the ResNet works on ImageNet, and running main.py in Pycharm. I just simply copy+paste the original code provided, but got the error:

"usage: scratch.py [-h] [-a ARCH] [-j N] [--epochs N] [--start-epoch N] [-b N]
                  [--lr LR] [--momentum M] [--wd W] [-p N] [--resume PATH]
                  [-e] [--pretrained] [--world-size WORLD_SIZE] [--rank RANK]
                  [--dist-url DIST_URL] [--dist-backend DIST_BACKEND]
                  [--seed SEED] [--gpu GPU] [--multiprocessing-distributed]
                  DIR
scratch.py: error: the following arguments are required: DIR

Process finished with exit code 2"

I'm pretty new at this stage, so any ideas or helps I do really appreciate.

**Comment by ludava** [comment]
at the end of your command that you run, instead of DIR, put the path to the folder containing your dataset (both train and val folders)
